tests/test_conf.py

```diff
@@ -87,6 +88,7 @@
     "post_bump_hooks": ["scripts/slack_notification.sh"],
     "prerelease_offset": 0,
     "encoding": "utf-8",
+    "always_signoff": False,
```

Comment:
Could we add a test case to verify that when `always_signoff` is set to true, we're actually running commit with signoff?